### DISCUSSION TITLE
Is there a way to change mouse scroll speed?

### DISCUSSION THREAD

**Discussion by xlucn** [qutebrowser/qutebrowser#6762] [Q&A] [answered]
By scroll speed I mean how much to scroll the page in one mouse wheel scroll action. I would like a solution without any external tools.

I can do it for keys with
```
bind j scroll-px 0 96
```
But don't know any way to configure that for mouse scroll.

Do I have to wait for #591 ?

**Comment by The-Compiler** [comment, marked as answer]
Mouse scrolling is handled by the underlying Chromium, not qutebrowser. It [seems to lack](https://bugs.chromium.org/p/chromium/issues/detail?id=521211) a way to configure scroll speed outside of system-wide configuration. From what I can see, the only workaround is reimplementing the entire scrolling logic (poorly) in JS like [some](https://github.com/DemonTPx/chrome-scroll-speed/blob/master/content.js) [plugins](https://github.com/Dedas/linux-scroll-speed-fix/blob/master/content.js) do (and you could probably replicate in a Greasemonkey script with some work). Or indeed for qutebrowser, it handling the scrolling events manually and mapping it to scroll requests in the API somehow. Both seem like quite a hack.

**Reply by xlucn** [reply]
Thanks for all the information. That chromium issue is making me really worried. The scripting way is beyond my reach. So for me, the mouse wheel is basically unusable now :)
